tests/backend/integration/api/projects/test_resources.py

```diff
@@ -259,14 +259,16 @@ def assert_project_response(project_response, expected_project, assert_type="ful
         assert expected_project.id == project_response["projectId"]
         # Since some of the fields are not returned in summary mode we need to skip them
         if assert_type != "summary":
-            assert geojson.is_valid(project_response["areaOfInterest"])
+            assert geojson.loads(
+                geojson.dumps(project_response["areaOfInterest"])
+            ).is_valid
             assert ["type", "coordinates"] == list(
                 project_response["areaOfInterest"].keys()
             )
             if assert_type == "notasks":
                 assert "tasks" not in project_response
             else:
-                assert geojson.is_valid(project_response["tasks"])
+                assert geojson.loads(geojson.dumps(project_response["tasks"])).is_valid
```

Comment:
I observed this earlier. What does this accomplish?

Reply:
`geojson.is_valid` is no longer a function.
The code could just as easily be written as `geojson.loads(json.dumps(project_response["tasks"])).is_valid`. Essentially, I'm converting a `json` object into a `geojson` object.